postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -0,0 +1,4753 @@
+/*
+ * psql - the PostgreSQL interactive terminal
+ *
+ * Copyright (c) 2000-2018, PostgreSQL Global Development Group
+ *
+ * src/bin/psql/command.c
+ */
```

Comment:
Update file name and tool name in the comment.

Reply:
Ok